solidity/test/random_beacon_operator/TestPricingRewards.js

```diff
@@ -46,7 +46,8 @@ describe('KeepRandomBeaconOperator/PricingRewards', function() {
     let entryFeeEstimate = await serviceContract.entryFeeEstimate(0);
     await serviceContract.methods['requestRelayEntry()']({value: entryFeeEstimate, from: accounts[0]});
 
-    mineBlocks((await operatorContract.relayEntryGenerationTime()).addn(1));
+    let relayEntryGenerationTime = await operatorContract.relayEntryGenerationTime();
```

Comment:
Just a heads up, we'll have a conflict with https://github.com/keep-network/keep-core/pull/1569 here.